### ISSUE TITLE
WebDriver BiDi: Expand list of people with admin merge permissions

### ISSUE THREAD

**Issue by thiagowfx** [web-platform-tests/wpt#40860]
Currently, due to some ongoing CI infra issues (e.g. [Chromium](https://github.com/web-platform-tests/wpt/issues/38450), for Firefox I am not sure which Issue to link to), most PRs that touch files under `/webdriver/tests/bidi` or `/webdriver/tests/support` fail, and thus **cannot be automatically merged**.

The problematic CI jobs are often the following two:

- `wpt-firefox-nightly-stability`. [Example failure](https://github.com/web-platform-tests/wpt/pull/40813/checks?check_run_id=14737114648). Example PR: https://github.com/web-platform-tests/wpt/pull/40813
- `wpt-chrome-dev-stability`. [Example failure](https://github.com/web-platform-tests/wpt/pull/40814/checks?check_run_id=14739750286). Example PR: https://github.com/web-platform-tests/wpt/pull/40814.

The workflow is such that one always needs to ask one of the current admins to admin merge the PRs. [Example](https://github.com/web-platform-tests/wpt/pull/40814#issuecomment-1618697877) (you'll find many more similar examples in recently merged PRs).

You can find more data points [here](https://github.com/web-platform-tests/wpt/issues/38450#issuecomment-1584656355).

**This is an abnormal workflow**
In fact, an RFC was event sent to make those jobs non-required (c.f. https://github.com/web-platform-tests/rfcs/pull/131) -- well, at least the Chromium one, not sure about Firefox.
Ideally the CI jobs should be stable / non-flaky, so that all checks would be green, so that e.g. **auto-merge** would work once at least one approval is attained, without the requirement that the approver needs to be an admin.

Given that this flaky state is the current status quo for a while, **I would like to ask that more people be added as admins**.

Currently, amongst the current admins **within the subset of people that actually work on webdriver bidi** (https://github.com/orgs/web-platform-tests/teams/admins/members && https://github.com/orgs/web-platform-tests/teams/wpt-core-team), there's at least one person from Apple (@gsnedders) and one person from Mozilla (@jgraham). There's also one person from Google (@foolip) who's currently on extended leave.

**My ask is: Can we add someone else from Google (who actively works on BiDi) as admin?** I am not sure what the bar is to add more admins. I don't mind volunteering myself to become one. However, if the bar is high, we could add someone else who has been around for a longer time (e.g. @mathiasbynens or @OrKoN).

In the meantime I've been asking @DanielRyanSmith to merge some of my PRs so that we reduce our [bus factor](https://en.wikipedia.org/wiki/Bus_factor), but this is not ideal and doesn't scale because although Daniel is from Google, he is not part of the BiDi team.

It's just that the current process to submit PRs from Google to WPT is quite cumbersome / slow and often takes O(days) when ideally it should take O(hours) for PRs that are not controversial, as we always need to wait for _at least_ two _serial_ reviews: one from a domain reviewer, and one from an admin (which only happens after the domain review is approved).
Fixing this gap would unlock the possibility to allow us to become more productive and iterate on more PRs more quickly.

**Comment by jrandolf** [comment]
+1 This is a major bottleneck for development.

**Comment by mathiasbynens** [comment]
> My ask is: Can we add someone else from Google (who actively works on BiDi) as admin? I am not sure what the bar is to add more admins. I don't mind volunteering myself to become one. However, if the bar is high, we could add someone else who has been around for a longer time (e.g. @mathiasbynens or @OrKoN).

FWIW, I would be happy to take on this role if deemed worthy. I’m colocated with the team at Google working on BiDi and already-domain-approved-PRs could be sent to me where appropriate.

**Comment by gsnedders** [comment]
> Currently, amongst the current admins (https://github.com/orgs/web-platform-tests/teams/admins/members && https://github.com/orgs/web-platform-tests/teams/wpt-core-team) within the subset of people that actually work on webdriver bidi, there's at least one person from Apple (@gsnedders) and one person from Mozilla (@jgraham). There's also one person from Google (@foolip) who's currently on extended leave.

I don't see why we should be limiting ourselves to people who are working on WebDriver BiDi? After all—for most areas of the web platform, no admin is a direct contributor.

The reality is we have 16 admins already, which I'd hope gave us a decent number with decent timezone coverage.

**Comment by thiagowfx** [comment]
Update: https://github.com/web-platform-tests/wpt/issues/40990 is now filed.

Minutes: https://github.com/web-platform-tests/wpt-notes/blob/master/minutes/2023-07-11.md#wpt-issue-40860---webdriver-bidi-expand-list-of-people-with-admin-merge-permissions

**Comment by thiagowfx** [comment]
@jgraham Is there a firefox bug we can follow regarding the current firefox stability job failure?

Is https://bugzilla.mozilla.org/show_bug.cgi?id=1832541 the correct bug?

**Comment by OrKoN** [comment]
I believe the stability bug is https://bugzilla.mozilla.org/show_bug.cgi?id=1841792

**Comment by whimboo** [comment]
> I believe the stability bug is https://bugzilla.mozilla.org/show_bug.cgi?id=1841792

And the Firefox stability is fixed now on this repo via https://github.com/web-platform-tests/wpt/pull/41243.

**Comment by whimboo** [comment]
Shall we close this issue given that expanding the list of admins is not what people want?

**Comment by thiagowfx** [comment]
Yes, fine to close it for the time being. We can reopen it in the future if needed.

**Comment by whimboo** [comment]
> The reality is we have 16 admins already, which I'd hope gave us a decent number with decent timezone coverage.

@gsnedders this doesn't seem to really work out. Beside jgraham (who is out atm), I tagged you and one other admin on a lot of PRs where admin merge is needed. Sadly so far no reply / action from anyone. Ok, I'm not sure if one of you two are off as well.

As such is there a mechanic for automatically tagging all the admins at once so that the one who sees it first can actually do the merge? I don't think it's helpful to have to CC each and every admin manually in case an admin merge is needed. Thanks.

**Comment by whimboo** [comment]
FYI over on https://github.com/web-platform-tests/wpt/pull/41661 I tried to CC the `@web-platform-tests/admins` team. Not sure if that is working, but maybe that's a solution?
